projects/packages/videopress/src/client/block-editor/hooks/use-preview.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * WordPress dependencies
+ */
+import { store as coreStore } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+/*
+ * Internal dependencies
+ */
+import { VideoPreview } from '../blocks/video/types';
```

Comment:
Usually, we use to add types in a `type` section, setting explicitly that it's a matter of types:
```suggestion
/*
 * Types
 */
import type { VideoPreview } from '../blocks/video/types';
```